finality-aleph/src/new_network/connection_manager/session.rs

```diff
@@ -30,10 +37,10 @@ pub enum AddressError {
 }
 
 async fn construct_authentication(
```

Comment:
Name is now confusing and it also looks like a merge of two unrelated functions.

Reply:
Idea for better name? @timorl what do you think since you designed it

Reply:
Uh, `construct_session_info` is a pretty clear candidate, innit?

Reply:
Done